scripts/build_imagemagick

```diff
@@ -13,56 +13,23 @@ function handle_error() {
 }
 trap handle_error EXIT SIGINT
 
-function add_xenial_repositories() {
-  cat <<-EOT >> /etc/apt/sources.list
-  deb http://archive.ubuntu.com/ubuntu/ xenial main universe
-  deb http://security.ubuntu.com/ubuntu xenial-security main
-EOT
-}
+# avoid error: E: You must put some 'deb-src' URIs in your sources.list
+echo "deb-src http://archive.ubuntu.com/ubuntu noble main universe" >> /etc/apt/sources.list
 
-# add source packages repository to sources.list
-function add_source_repositories() {
-  echo "deb-src http://archive.ubuntu.com/ubuntu/ xenial main universe" >> /etc/apt/sources.list
-  if [[ $STACK == "heroku-20" ]]; then
-    add_xenial_repositories
-  fi
-}
 
-add_source_repositories
 
 echo "⚙️ Getting libraries and dependencies..."
 apt-get update -y
 apt-get install -y build-essential autoconf libtool git-core
-if [[ $STACK == "heroku-20" ]]; then
-  echo "⚙️ Getting additional dependencies for the selected stack..."
-  apt-get install -y libomp5 libomp-10-dev libjasper-dev
-fi
 apt-get build-dep -y imagemagick libmagickcore-dev
 
-# install the libde265 library
-echo "⚙️ Installing and configuring libde265..."
-git clone https://github.com/strukturag/libde265.git $WORK_DIR/libde265
-cd $WORK_DIR/libde265
-./autogen.sh && ./configure && make && make install
-
-# install the libheif library
-echo "⚙️ Installing and configuring libheif..."
-git clone https://github.com/strukturag/libheif.git $WORK_DIR/libheif
-cd $WORK_DIR/libheif
-./autogen.sh && ./configure && make && make install
-
-# get, configure and install newest imagemagick
-echo "⚙️ Installing latest imagemagick..."
-git clone https://github.com/ImageMagick/ImageMagick.git $WORK_DIR/imagemagick
+# get, configure and install imagemagick
+echo "⚙️ Installing imagemagick version 7.0.11-5..."
+git clone --depth 1 --branch 7.0.11-5 https://github.com/ImageMagick/ImageMagick.git $WORK_DIR/imagemagick
```

Comment:
`---depth 1` - don't clone the whole repo.

`--branch 7.0.11-5` - only clone the tag "7.0.11-5".